spec/models/task_spec.rb

```diff
@@ -1,5 +1,23 @@
 require 'rails_helper'
 
-RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe 'バリデーション確認', type: :model do
+  describe 'validation' do
+    let(:task){ FactoryBot.create(:task, title: 'title1', status: 'todo')}
+
+    it 'バリデーションが通っていること' do
+      expect(task).to  be_valid
+    end
+
+    it 'タイトルが入力されていること' do
+      task.title = nil
+      expect(task).to_not be_valid
```

Comment:
こちらは好みですがこの様にも記載できます
```suggestion
      expect(task).to be_invalid
```